### PR TITLE
WIP: changes for target independent build

### PR DESCRIPTION
**Pull request by glennchid**
Target specific boot sources (FSBL/boot.bin, devicetree)  moved to FPGA repo.
NB. Merge should only be made when corresponding change made in other repo.

Still to do:
- led deamon

### CODE REVIEW

**Comment by Araneidae** [comment]
Looks like there are some merge conflicts that will need to be resolved first.

File: Makefile [hunk 1]
```diff
@@ -36,6 +36,29 @@ KERNEL_TAG = xilinx-v2015.1
...
+#    UBOOT_CONFIG = zynq_zc70x_config
+#else ifeq($(PLATFORM,zynqmp)
+#    ARCH=aarch64
+#    COMPILER_PREFIX=aarch64-linux-gnu-
+#    # From Vivado 2020 onwards we can use the common defconfig
+#    #UBOOT_CONFIG = xilinx_zynqmp_virt_defconfig
+#    # For earlier Vivado verions, we can try zcu102 as an initial guess for the relevant config.
+#    UBOOT_CONFIG = xilinx_zynqmp_zcu102_rev1_0_defconfig
+#else
+#    $$(error Unknown PLATFORM specified. Must be 'zynq' or 'zynqmp')
```
Comment (Araneidae) [inline Makefile:61]: I'm really not hugely happy with this big block of code here.
1. It's commented out but substantial
2. It's target specific, so shouldn't be in this top level file
Can this be placed in a separate file and either made real or not include in this merge.

**Review by Araneidae — CHANGES_REQUESTED** [review]
So it looks as if this merge simply removes the boot files.  Without the extra target specific settings in `Makefile` I'm happy to merge this.

**Comment by thomascobb** [comment]
Superseded by master

**Pull request closed without merging**